docs/usage/CART_DISCOUNT_SYNC.md

```diff
@@ -25,11 +25,19 @@ against a [CartDiscountDraft](https://docs.commercetools.com/http-api-projects-c
 ### Sync list of cart discount drafts
 
 #### Prerequisites
-1. The sync expects a list of `CartDiscountDraft`s that have their `key` fields set to be matched with
+1. Create a `sphereClient`:
+Use the `ClientConfigurationUtils#createClient` util which applies the best practices for `SphereClient` creation.
+
+    If you have special requirements on the sphere client creation, then you can use the following:  
+    - Limit the number of concurrent requests done to CTP. This can be done by decorating the `sphereClient` with [QueueSphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/QueueSphereClientDecorator.html)
+ 
+    - Retry decorator on 5xx errors with a retry strategy. This can be achieved by decorating the `sphereClient` with the [RetrySphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/RetrySphereClientDecorator.html)
+
```

Comment:
Do you think it would be better to link to a separate page which describes both decorators and maybe more in the future instead of repeating it everywhere?

Reply:
Until now this information was being linked. It was included in the "Important Usage Tips" page. The goal of this change is to improve the visibility of this information, that is why it is now included in every user guide. However, since I am not a user of the library I am not sure how users are using the docs and where they would search for this information.

Reply:
2f5557f4